frontend/src/component/Navigation.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./Navigation.css"
+import {NavLink} from "react-router-dom";
+
+export default function Navigation() {
+
+    return (
+        <nav>
+            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+            <div className="navbar">
+                <NavLink to="/" className="active"><i className="fa fa-fw fa-home"></i> Home</NavLink>
+                <a href="#"><i className="fa fa-fw fa-user"></i> Login</a>
```

Comment:
This link doesn't have any functionality? There is no href and no onClick specified

Reply:
Tha link pick up the icons for the nav bar 👍